src/map.rs

```diff
@@ -664,8 +687,7 @@ where
     #[cfg_attr(feature = "inline-more", inline)]
     pub fn reserve(&mut self, additional: usize) {
         let hash_builder = &self.hash_builder;
-        self.table
-            .reserve(additional, |x| make_hash(hash_builder, &x.0));
+        self.table.reserve(additional, make_hasher(hash_builder));
```

Comment:
```suggestion
        self.table.reserve(additional, make_hasher(&self.hash_builder));
```
The `let` above was only needed because the closure would otherwise capture all of `self` instead of just the `hash_builder` field.